Mage.Sets/src/mage/cards/j/JabarisInfluence.java

```diff
@@ -0,0 +1,102 @@
+
+/**
+ * @author Cguy7777
+ */
+public final class JabarisInfluence extends CardImpl {
+
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent("nonartifact, nonblack creature that attacked you this turn");
+
+    static {
+        filter.add(JabarisInfluencePredicate.instance);
+    }
+
+    public JabarisInfluence(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{3}{W}{W}");
+
+        // Cast Jabari's Influence only after combat.
+        this.addAbility(new CastOnlyDuringPhaseStepSourceAbility(
+                null, null, AfterCombatCondition.instance, "Cast this spell only after combat"));
+
+        // Gain control of target nonartifact, nonblack creature that attacked you this turn and put a -1/-0 counter on it.
+        this.getSpellAbility().addEffect(new GainControlTargetEffect(Duration.Custom));
+        this.getSpellAbility().addEffect(new AddCountersTargetEffect(CounterType.M1M0.createInstance()).setText("and put a -1/-0 counter on it"));
+        this.getSpellAbility().addTarget(new TargetCreaturePermanent(filter));
+        this.getSpellAbility().addWatcher(new JabarisInfluenceWatcher());
+    }
+
+    private JabarisInfluence(final JabarisInfluence card) {
+        super(card);
+    }
+
+    @Override
+    public JabarisInfluence copy() {
+        return new JabarisInfluence(this);
+    }
+}
+
+enum JabarisInfluencePredicate implements ObjectSourcePlayerPredicate<Permanent> {
+    instance;
+
+    @Override
+    public boolean apply(ObjectSourcePlayer<Permanent> input, Game game) {
+        Permanent permanent = input.getObject();
+
+        JabarisInfluenceWatcher watcher = game.getState().getWatcher(JabarisInfluenceWatcher.class);
+        return watcher.checkIfEligibleCreatureAttackedYouThisTurn(permanent, input.getPlayerId(), game);
+    }
+}
+
+class JabarisInfluenceWatcher extends Watcher {
+
+    private final Map<MageObjectReference, UUID> eligibleAttackedThisTurnCreatures = new HashMap<>();
+
+    public JabarisInfluenceWatcher() {
+        super(WatcherScope.GAME);
+    }
+
+    @Override
+    public void watch(GameEvent event, Game game) {
+        if (event.getType() == GameEvent.EventType.ATTACKER_DECLARED) {
+            Permanent permanent = game.getPermanent(event.getSourceId());
+            if (permanent != null && !permanent.isArtifact(game) && !permanent.getColor(game).equals(ObjectColor.BLACK)) {
+                eligibleAttackedThisTurnCreatures.put(new MageObjectReference(event.getSourceId(), game), event.getTargetId());
+            }
+        }
+    }
+
+    boolean checkIfEligibleCreatureAttackedYouThisTurn(Permanent permanent, UUID playerId, Game game) {
+        MageObjectReference mor = new MageObjectReference(permanent, game);
+        return eligibleAttackedThisTurnCreatures.containsKey(mor) && eligibleAttackedThisTurnCreatures.get(mor).equals(playerId);
+    }
+
+    @Override
+    public void reset() {
```

Comment:
miss `super.reset()`

Reply:
Thanks for catching that. I based some of this off of Gideon's Triumph, which also [doesn't call super.reset() in its reset()](https://github.com/magefree/mage/blob/e449282c8f02bc9c6aaa8e0109c7ac766df8223c/Mage.Sets/src/mage/cards/g/GideonsTriumph.java#L116C8-L116C8). Perhaps that should be changed as well?

Reply:
Yeah I presume that's an oversight in Gideon's Triumph. Admittedly it shouldn't matter because all the super does is `condition = false;` and not all watchers use the `condition` field. I just think it's good practice to keep things working as expected in case somebody later adjusts the usage.